src/runtime_src/core/pcie/driver/linux/xocl/devices.h

```diff
@@ -1482,6 +1482,12 @@ struct xocl_subdev_map {
 	 		.name = NODE_STM4,		\
 			.flags = IORESOURCE_MEM,	\
 		},					\
+		{					\
+			.start = 0x3000000,		\
+			.end = 0x3008000,		\
+	 		.name = NODE_QDMA4_CSR,		\
+			.flags = IORESOURCE_MEM,	\
+		},					\
```

Comment:
Why do we need static define of XOCL_RES_QDMA4? Is QDMA4 only for 2RP platforms?

Reply:
Hmm, I am not aware of EQDMA for 1RP.